website/src/components/Layout/ChatLayout.tsx

```diff
@@ -9,7 +9,7 @@ import { SideMenuItem } from "../SideMenu";
 import { ToSWrapper } from "../ToSWrapper";
 
 export const ChatLayout = ({ children }: PropsWithChildren) => (
-  <div className="min-h-screen h-screen max-h-screen flex flex-col overflow-hidden">
+  <div className="min-h-screen-dvh h-screen-dvh max-h-screen-dvh flex flex-col overflow-hidden">
```

Comment:
could you please use this?
https://github.com/tailwindlabs/tailwindcss/discussions/8216#discussioncomment-4621818

We don't need a new plugin.